settings.gradle

```diff
@@ -7,6 +7,8 @@ include 'jetty'
 include 'kafka'
 include 'logback'
 include 'retrofit2'
+include 'spring-boot-autoconfigure'
```

Comment:
Ah just noticed the directory structure - what do you think about spring-boot/autoconfigure and spring-boot/starter? I think you can override the name here for other build scripts to work